### PR TITLE
FileZillaProject.FileZillaServer version beta 0.9.60

### PR DESCRIPTION
**Pull request by zephiowl**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/20358)

### CODE REVIEW

File: manifests/f/FileZillaProject/FileZillaServer/beta 0.9.60/FileZillaProject.FileZillaServer.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: FileZillaProject.FileZillaServer
```
Comment (OfficialEsco) [inline manifests/f/FileZillaProject/FileZillaServer/beta 0.9.60/FileZillaProject.FileZillaServer.installer.yaml:4]:
```suggestion
PackageIdentifier: TimKosse.FileZillaServer
```

File: manifests/f/FileZillaProject/FileZillaServer/beta 0.9.60/FileZillaProject.FileZillaServer.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: FileZillaProject.FileZillaServer
```
Comment (OfficialEsco) [inline manifests/f/FileZillaProject/FileZillaServer/beta 0.9.60/FileZillaProject.FileZillaServer.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: TimKosse.FileZillaServer
```

File: manifests/f/FileZillaProject/FileZillaServer/beta 0.9.60/FileZillaProject.FileZillaServer.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: FileZillaProject.FileZillaServer
```
Comment (OfficialEsco) [inline manifests/f/FileZillaProject/FileZillaServer/beta 0.9.60/FileZillaProject.FileZillaServer.yaml:4]:
```suggestion
PackageIdentifier: TimKosse.FileZillaServer
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
This should be moved to Manifests\t\TimKosse\FileZillaServer so its in the same folder as FileZillaClient.

**Comment by Xanthus58** [comment]
> .

Ahhh thank you I didn't actually think to check that; my apologies.

**Comment by OfficialEsco** [comment]
Remember you have to manually move the files and rename the file.

**Comment by Xanthus58** [comment]
https://github.com/microsoft/winget-pkgs/pull/20360 

Made a new pull request with the suggested changed by OfficialEsco

**Pull request closed without merging**